tuna/worker_interface.py

```diff
@@ -112,7 +113,6 @@ def __init__(self, **kwargs):
     self.compile_only = False
     self.docker_name = TUNA_DOCKER_NAME
     self.gpu = None
-    self.bin_cache = False
```

Comment:
i think we can also remove self.find_db = None. that is set in the self.dbt structure. Double check that no other tables are set here.

Reply:
removing